aspnetcore/fundamentals/middleware/extensibility/samples/2.x/MiddlewareExtensibilitySample/README.md

```diff
@@ -1,10 +1,10 @@
 # ASP.NET Core Middleware Extensibility Sample
 
-This sample demonstrates the scenarios described in [Factory-based middleware activation in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/middleware/middleware-extensibility).
+This sample demonstrates the scenarios described in [Factory-based middleware activation in ASP.NET Core](https://learn.microsoft.com/aspnet/core/fundamentals/middleware/middleware-extensibility).
```

Comment:
Adding to broken link tab, try https://learn.microsoft.com/aspnet/core/fundamentals/middleware/extensibility

Reply:
Pre-existing issue, keeping for now.